lesson02/assignment/css/style.scss

```diff
@@ -2,5 +2,44 @@
 @import 'base.normalize';
 @import 'settings.variables';
 @import 'settings.responsive';
+@import 'components.grid';
 @import 'components.content';
 
+
+// general style rules for unordered list and headlines
+h1 {
+    color: $dark-gray;
+    font-size: 1.5em;
+}
+
+h2 {
+    color: $gray;
+    font-size: 1em;
+    font-weight: 100;
+    font-style: italic;
+}
+
+ul {
+    padding-left: 0;
+    list-style-type: none;
+    line-height: 2.5em;
+    font-size: .8em;
```

Comment:
The html has a ul tag nested inside a ul tag, so the .8em rule is applied twice, resulting in a font size of 10.24px (computed) instead of the likely intended 12.8px.

The fastest fix would be to use rems here instead of ems. Alternatively, you can apply the font size rule with ems to the grid-list class.

Reply:
Got it. Thanks!